runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -431,6 +431,18 @@ else if (config.swapHomePortal() != HouseMode.ENTER && option.equals("enter"))
 					break;
 			}
 		}
+		else if (config.swapTeleportToDestination() != ObeliskMode.ACTIVATE)
```

Comment:
???

This should be `target.equals("obelisk")`, otherwise it will try to swap entries on all clicks.

Reply:
Ok, thanks. I'm new to this, thanks for helping me out. If anything else needs to be changed, let me know.